manywheel/build_docker.sh

```diff
@@ -52,16 +52,30 @@ DOCKER_IMAGE=${DOCKER_REGISTRY}/pytorch/manylinux-builder:${DOCKER_TAG}
         "${TOPDIR}"
 )
 
+GITHUB_REF=${GITHUB_REF:-}
+GIT_BRANCH_NAME=${GITHUB_REF##*/}
+GIT_COMMIT_SHA=${GITHUB_SHA:-}
+DOCKER_IMAGE_BRANCH_TAG=${DOCKER_IMAGE}-${GIT_BRANCH_NAME}
+DOCKER_IMAGE_SHA_TAG=${DOCKER_IMAGE}-${GIT_COMMIT_SHA}
+
```

Comment:
Can we set the default values to something that we can derive locally here?

i.e. `GITHUB_REF=$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match)`

```suggestion
GITHUB_REF=${GITHUB_REF:$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match)}
GIT_BRANCH_NAME=${GITHUB_REF##*/}
GIT_COMMIT_SHA=${GITHUB_SHA:-$(git rev-parse HEAD)}
DOCKER_IMAGE_BRANCH_TAG=${DOCKER_IMAGE}-${GIT_BRANCH_NAME}
DOCKER_IMAGE_SHA_TAG=${DOCKER_IMAGE}-${GIT_COMMIT_SHA}

```

Just to make it so we can run this locally as well and don't have to rely on the env variables being present

Reply:
done!